core/tests/CMakeLists.txt

```diff
@@ -34,5 +35,6 @@ endif()
 
 if(${run_e2e_tests})
     add_subdirectory(gateway_e2e)
+    add_subdirectory(performance_e2e)
```

Comment:
I'm going to leave it as e2e for now - if we get more tests like this, then it might make more sense to break them into their own category.